SMLHelper/Handlers/OptionsPanelHandler.cs

```diff
@@ -16,6 +16,7 @@ public class OptionsPanelHandler
         public static void RegisterModOptions(ModOptions options)
         {
             OptionsPanelPatcher.modOptions.Add(options);
+            OptionsPanelPatcher.modOptions.Sort();
```

Comment:
This is fine for now, but if the idea is to sort on insert, we might as well be using change the `modOptions`  List into a [SortedList](https://docs.microsoft.com/en-us/dotnet/api/system.collections.sortedlist?view=netframework-4.7.2) type.

And then this would look more like this
```CSharp
OptionsPanelPatcher.modOptions.Add(options.Name, options);
```
With no actual need to add the `IComparable` interface to the `ModOptions` class.

Reply:
Addressed in commit 0be7232